src/scenarios/Modals/Modals.module.scss

```diff
@@ -1,5 +1,7 @@
 @use 'assets/styles/Variables.scss';
 
+// Common styling for modals
```

Comment:
Det er good at denne kommentaren står?

Reply:
Ja tenker det, la den til fordi vi ikke deler en styling-fil så ofte i applikasjonen. Liten warning om at hvis man forandrer noe her blir det forandret for flere komponenter.